### DISCUSSION TITLE
Calendar button in mail app

### DISCUSSION THREAD

**Discussion by GoCho85** [tutao/tutanota#7660] [Ideas]
Just downloaded the calendar app and was curious how the integration with the mail app is working.

Maybe this could be an idea: when the calendar app is installed, the button "calendar" should neither work or opens the calendar app.
I know some other apps can do this, maybe this could help here, because the calendar app has a better functionality.

Cheers

**Comment by Gitchick57** [comment]
Agreed, makes sense.



Oct 3, 2024, 4:48 AM by ***@***.***:

**Comment by serrq** [comment]
Removing the calendar tab in the Tuta Mail app. It is not necessary now, and free the space for the upcoming "Tuta Drive" and "Tuta Note" tabs.

The «button» in Tuta Mail will leads to three scenarios:

1. The user has Tuta Cal installed, so "go to the Cal app".
2.  The user haven't the Cal app installed. A popup leads to tuta.com/download with all installation options available.
3. Open the Tuta Cal web app
